pokeapp/src/utilities/ApiCall.js

```diff
@@ -1,12 +1,17 @@
 const Pokedex = require("pokeapi-js-wrapper");
-const P = new Pokedex.Pokedex();
 
 const interval = {
   offset: 0,
   limit: 1119,
-  cache: true,
 };
 
+const customOptions = {
+  cache: true,
+}
```

Comment:
This should include more than just a cache flag.  Please see this doc for more info:  https://github.com/PokeAPI/pokeapi-js-wrapper#configuration